proto/umee/incentive/v1/incentive.proto

```diff
@@ -0,0 +1,84 @@
+syntax = "proto3";
+package umeenetwork.umee.incentive.v1;
+
+import "cosmos/base/v1beta1/coin.proto";
+import "gogoproto/gogo.proto";
+
+option go_package = "github.com/umee-network/umee/v3/x/incentive";
+
+// Params defines the parameters for the incentive module.
+message Params {
+  option (gogoproto.equal) = true;
+
+  // max_unbondings defines the maximum amount of concurrent unbondings an address can have.
+  uint32 max_unbondings = 1;
+
+  // unbonding_duration_long defines the unbonding duration (in seconds) of the long tier.
+  uint64 unbonding_duration_long = 2;
+
+  // unbonding_duration_middle defines the unbonding duration (in seconds) of the middle tier.
+  uint64 unbonding_duration_middle = 3;
+
+  // unbonding_duration_short defines the unbonding duration (in seconds) of the short tier.
+  uint64 unbonding_duration_short = 4;
+
+  // The tier_weight_short defines how the proportion of rewards assets locked
+  // in the short unbonding duration receive compared to what the same amount
+  // would receive on the long tier.
+  string tier_weight_short = 5 [
+    (gogoproto.customtype) = "github.com/cosmos/cosmos-sdk/types.Dec",
+    (gogoproto.nullable)   = false
+  ];
+
+  // The tier_weight_middle defines how the proportion of rewards assets locked
+  // in the middle unbonding duration receive compared to what the same amount
+  // would receive on the long tier.
+  string tier_weight_middle = 6 [
+    (gogoproto.customtype) = "github.com/cosmos/cosmos-sdk/types.Dec",
+    (gogoproto.nullable)   = false
+  ];
+}
+
+// IncentiveProgram defines a liquidity mining incentive program on a single
+// locked uToken denom that will run for a set amount of time.
+message IncentiveProgram {
+  option (gogoproto.equal) = true;
+
+  // ID uniquely identifies the incentive program after it has been created.
+  // It is zero when the program is being proposed by governance.
+  uint32 id = 1;
+
+  // start_time is the block time at which the incentives begin.
```

Comment:
```suggestion
  // start_time is a timestamp (in seconds) at which the incentives begin.
```

Reply:
We don't know the block time in advance. So it should be either a timestamp or a block height.